src/lib/components/input/toggle/index.tsx

```diff
@@ -0,0 +1,118 @@
+import classNames from "classnames";
+
+import styles from './styles.module.scss';
+export interface ToggleWithDescription {
+  title: string;
+  description?: string;
+}
+
+export interface ToggleProps<ValueType extends string> {
+  options: Record<ValueType, string | ToggleWithDescription>;
```

Comment:
Should we update `string` to `ReactNode` to align it with the checkbox/radio buttons or not for now? It might create more confusion - I associate toggles with switching between boolean values but we're not meant to use them here.

Reply:
I was thinking about doing this after your work but I think I can copy your code here and just be done with this 😄